incubator/kafqa/templates/telegraf.configmap.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+data:
+  telegraf.conf: |-
+    [global_tags]
+      cluster_name = {{ .Values.environment | quote }}
+    [agent]
+      collection_jitter = "0s"
```

Comment:
enable debug log and add logfile

Reply:
It's good to have default value as false here. 

Log file name, the empty string means to log to stderr.
Logfile = ""
For kubernetes, its better to log in the console right ?